.github/workflows/docker-image.yml

```
@@ -0,0 +1,45 @@
+name: "Docker Image"
+
+on:
+  push:
+    branches:
+      - "main"
```

You can add the schedule here:

```suggestion
  workflow_dispatch:
  schedule:
    # At 06:01 on the 5th of every month.
    - cron: '1 6 5 * *'
  push:
    branches:
      - "main"
```